### PR TITLE
Merging M4 restaurants like dislike into Moot Branch

### LINKED ISSUE #138: Render Restaurants From Database
**Point Value:** 1 point
**Size Category:** Small
**Description:** Use stored data in database to render restaurant cards instead of the Restaurants.csv file; make POST request to database in the card swipe listener function to render the next restaurant card from db on click (like/dislike).

### PR DESCRIPTION
**Pull request by addie-p**
Merging updated databases into placeholder branch to integrate changes so far without accidentally breaking the Main branch. 
Changes:
Server linked to frontend (runs using "node Server.js" for backend integration)
Models for restaurants database initialized
Database with restaurant data fields created
CSV parsed and inserted into database using script
Database data pulled and retrieved using Get request
Test file created to test whether data is being correctly pulled/formatted

Adding reviewers to ensure good Github practices.

### CODE REVIEW

**Comment by evc29** [comment]
This looks very good, works properly under the /getrestaurants route! Restaurant data is properly formatted in GET request.

**Review by evc29 — APPROVED** [review]
This looks very good, works properly under the /getrestaurants route! Restaurant data is properly formatted in GET request.

File: .DS_Store
Comment (jadatwo) [inline .DS_Store:1]: need to discard later

**Review by SandraV255 — APPROVED** [review]
Changes look great: server runs correctly, restaurant data can be retrieved from database.

**Review by jadatwo — APPROVED** [review]
restaurant data successfully retrieved, works properly

**Pull request merged**